apps/dashboard/app/helper.ts

```diff
@@ -0,0 +1,14 @@
+import { colorMap } from "@/components/transaction-details/index.types";
```

Comment:
the filename (helper.ts) is too generic and not helpful / a good representation of what this function is doing